src/bflib_netsp.cpp

```diff
@@ -264,7 +264,7 @@ TbError ServiceProvider::Send(unsigned long plr_id, void *buf)
       break;
   case NETMSGTYPE_ADD:
       memcpy(&p3, (uchar *)buf+4, sizeof(unsigned long));
-      strncpy(str,(char *)buf+8, 32);
+      snprintf(str,32, "%s", (char*)buf + 8);
```

Comment:
what is 32? Douglas Adams praised 42!

Reply:
I don't know. Do you?

Reply:
32 is the size of `str`. Can replace it with `sizeof(str)`.

Reply:
ok